ckb-bin/src/subcommand/init.rs

```diff
@@ -184,7 +189,16 @@ pub fn init(args: InitArgs) -> Result<(), ExitCode> {
         }
     } else if args.chain == "dev" {
         println!("create {}", SPEC_DEV_FILE_NAME);
-        Resource::bundled(SPEC_DEV_FILE_NAME.to_string()).export(&context, &args.root_dir)?;
+        let bundled = Resource::bundled(SPEC_DEV_FILE_NAME.to_string());
+        if args.use_default_spec {
```

Comment:
I think the priority of `customize_spec` is greater than `use_default_spec`. When a user provides both `customize_spec` and `use_default_spec`, it should apply `customize_spec` or fail.

If we apply `customize_spec`:

```
if !args.customize_spec.is_unset() {
  ..
} else if args.use_default_spec {
  ..
}
```

Reply:
The default mode is "customize", that is why the priority of "customize" is less than "default".

Reply:
If two options are conflicting, use a exclusive options group instead.

Reply:
Already done in cli.rs